_data/guides_and_templates.yml

```
@@ -21,3 +21,12 @@
       text: LaTeX document templates
     - url: https://github.com/cnerg/py-template
       text: Python project template
+  - title: CHTC
```

indentation causing an error
```suggestion
- title: CHTC
```